advice/advice.js

```diff
@@ -0,0 +1,19 @@
+    /*************** Back Icon *********************/
+         function backIcon(){
+             window.location.href ="../symptoms/Symptoms.html";
+         }
+
+    /***************Dark Mode***********************/
+        var icon = document.getElementById("icon");
```

Comment:
Try to avoid using var as you can, its preferable to use const here instead of var